src/content/actions/receive-profile.js

```diff
@@ -393,8 +393,11 @@ export function retrieveProfileFromWeb(hash: string): ThunkAction {
     dispatch(waitingForProfileFromWeb());
 
     try {
+      const url = hash.startsWith('http')
```

Comment:
Please check the dataSource instead. I'd prefer not to support the URLs `/public/encodedurl/` or `/from-url/profilestorehash/`.